spec/lib/api/login/login-spec.js

```diff
@@ -260,9 +264,14 @@ describe('Http.Api.Login', function () {
         });
 
         after('stop server, restore mock and configure',function () {
-            cleanUp();
-            //restore endpoint
-            endpoint.authEnabled = true;
+            return cleanUp().then(function(){
+                return Promise.resolve()
```

Comment:
set the `endpoint.authEnabled` directly in `then`, no need to wrap another `Promise`